client/src/app/game/soundButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react";
+import css from "./soundButton.module.css";
```

Comment:
I use dashes for word separation in all files names, please rename all of them to `sound-button.something`